newapi/ooniapi/measurements.py

```diff
@@ -484,7 +495,14 @@ def get_measurement_meta():
     if not full:
         return cachedjson(24, **msmt_meta)
 
-    body = _fetch_measurement_body(report_id, input)
+    try:
+        body = _fetch_measurement_body(report_id, input, msmt_meta["measurement_uid"])
+        assert isinstance(body, bytes)
+        body = body.decode()
+    except Exception as e:
+        log.error(e, exc_info=1)
+        body = ""
```

Comment:
Shouldn't we raise an exception rather than returning an empty body here?

Reply:
No, it's what Explorer expects